src/utils/node/pm2.ts

```diff
@@ -0,0 +1,129 @@
+import * as path from 'path';
+import { connect, describe, disconnect, list, ProcessDescription, restart, start, stop } from 'pm2';
+
+const connectPM2 = async (): Promise<void> =>
+    new Promise<void>((resolve, reject) => {
+        connect(err => {
+            if (err) {
+                reject(err);
+
+                return;
+            }
+            resolve();
+        })
+    })
+
+const startPM2 = async (installPath: string, name: string): Promise<void> =>
+    new Promise<void>((resolve, reject) => {
+        start({
+            name,
```

Comment:
I like your idea, but the downside of defining this pm2 config directly here will have issues when the core pm2 make changes, for example, the `app.js` thing or any other pm2 configuration changes. So, for every change in core we need to create a new release for the commander, for now, it should be fine, but let's discuss what would be the better approach for longer term.

Reply:
yea probably we should change the pm2 config to read from the core
we should move the pm2 config to the `core` =)